core/src/main/java/com/novoda/noplayer/internal/exoplayer/ResumeableAdverts.java

```diff
@@ -0,0 +1,37 @@
+package com.novoda.noplayer.internal.exoplayer;
+
+import com.google.android.exoplayer2.C;
+import com.google.android.exoplayer2.source.ads.AdPlaybackState;
+
+final class ResumeableAdverts {
+
+    private ResumeableAdverts() {
+        // Utility class.
+    }
+
+    static AdPlaybackState markAsResumeableFrom(AdPlaybackState adPlaybackState, long positionInMillis) {
```

Comment:
Moved to a separate class because it felt like an additional operation **after** the initial creation, also helped with testing.